src/@types/events.ts

```diff
@@ -6,6 +6,7 @@ enum Event {
     ANNOTATION_REMOVE = 'annotations_remove',
     ANNOTATIONS_INITIALIZED = 'annotations_initialized',
     ANNOTATIONS_MODE_CHANGE = 'annotations_mode_change',
+    ANNOTATIONS_STAGED_CHANGE = 'annotations_staged_change',
```

Comment:
Should this be `CREATOR_STAGED_CHANGE`? It's not directly related to the annotations CRUD operations. It also leaves room for status change events in the future, if we want them.